pages/common/cargo-clippy.md

```diff
@@ -21,12 +21,12 @@
 
 - Treat warnings as errors:
 
-`RUSTFLAGS="-Dwarnings" cargo clippy -- -D warnings`
+`cargo clippy -- --deny warnings`
 
 - Run checks and ignore warnings:
 
-`cargo clippy -- -A warnings`
+`cargo clippy -- --allow warnings`
 
 - Apply Clippy suggestion automatically (experimental and only supported on the nightly channel):
 
-`cargo clippy --fix -Z unstable-options`
+`cargo clippy --fix --allow-features unstable-options`
```

Comment:
You misread the help page:

> -Z allow-features         -- Allow *only* the listed unstable features
> -Z timings                -- Display concurrency information
> -Z unstable-options       -- Allow the usage of unstable options

The `-Z` doesn't have a long argument and `allow-features` is an argument option, I think we should use token syntax here:
```suggestion
`cargo clippy --fix -Z {{unstable-options}}`
```

Reply:
The `fix` command now seems a stable feature.
See https://github.com/rust-lang/rust-clippy#automatically-applying-clippy-suggestions.

Reply:
So I added this commit: 99ced2f1.
What do you think?